src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -5572,6 +5572,39 @@ _pbs_python_event_set(unsigned int hook_event, char *req_user, char *req_host,
 			}
 		}
 
+	} else if (hook_event == HOOK_EVENT_RESV_END) {
+		struct rq_manage *rqj = req_params->rq_manage;
+
+		/* initialize event param to None */
+		(void)PyDict_SetItemString(py_event_param, PY_EVENT_PARAM_RESV,
+		Py_None);
+
+		if (IS_PBS_PYTHON_CMD(pbs_python_daemon_name)) {
+			if (py_pbs_statobj != NULL) {
+				Py_XDECREF(py_jargs);  /* discard previously used value */
+				py_rargs = Py_BuildValue("(sss)", "job", rqj->rq_objname,
+					pbs_conf.pbs_server_name); /* NEW ref */
+				py_resv = PyObject_Call(py_pbs_statobj, py_rargs,
+					NULL);/*NEW*/
```

Comment:
Is NEW a NEW ref?

Reply:
The NEW is actually a new object created for python reservation object.

Reply:
Can you be explicit by what you mean by NEW in the comment.  It will help people reading the code.